eng/pipelines/templates/publish-assets-and-packages.yml

```diff
@@ -143,7 +143,7 @@ jobs:
       allowPackageConflicts: true
 
   # Publishes the NuGet packages to DevDiv/VS (https://dev.azure.com/DevDiv/DevDiv/_artifacts/feed/VS)
-  # The RoslynInsertionTool will republish these packages to DevDiv/VS-CoreXtFeeds (https://dev.azure.com/DevDiv/DevDiv/_artifacts/feed/VS-CoreXtFeeds)
+  # A separate process will republish these packages to DevDiv/VS-CoreXtFeeds (https://dev.azure.com/DevDiv/DevDiv/_artifacts/feed/VS-CoreXtFeeds)
```

Comment:
This is accurate as I don't know the process that is involved, but it is still functioning, independent of us (no longer) using RoslynInsertionTool.